src/models/batch_rewards_schema.py

```diff
@@ -0,0 +1,38 @@
+"""Model for Batch Rewards Data"""
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any
+
+from pandas import DataFrame
+
+
+@dataclass
+class BatchRewards:
+    """
+    This class provides a transformation interface for the Dataframe we fetch from the orderbook
+    """
+
+    @classmethod
+    def from_pdf_to_dune_records(cls, rewards_df: DataFrame) -> list[dict[str, Any]]:
+        """Converts Pandas DataFrame into the expected stream type for Dune"""
+        return [
+            {
+                "block_number": int(row["block_number"]),
+                "tx_hash": row["tx_hash"],
+                "solver": row["solver"],
+                "data": {
+                    "reward_eth": float(row["reward_eth"]),
```

Comment:
Currently the per order rewards are in COW as floats because that is how we receive them from the DB. I suppose this was just a copy-pasta issue since the setup is so similar.